src/components/RequireAuth.js

```diff
@@ -204,12 +209,22 @@ const RequireAuth = (props) => {
   const modulesManager = useModulesManager();
   const auth = useAuthentication();
   const cfg = children.props.modulesManager.cfg;
+  const calendarSwitch = modulesManager.getConf(
+    "fe-core",
+    "allowSecondCalendar",
+    true,
+  );
+
   const isAppBarMenu = useMemo(() => theme.menu.variant.toUpperCase() === "APPBAR", [theme.menu.variant]);
+  const defaultValue = false;
+  const [isSecondaryCalendar, setSecondaryCalendar] = useBoolean(defaultValue);
```

Comment:
There is no need to declare seperate const for default value, just remove _defaultValue_ and do like this:
`const [isSecondaryCalendar, setSecondaryCalendar] = useBoolean(false);`